pkg/users/local.js

```diff
@@ -81,26 +83,38 @@
     );
 
     if (path.length === 0) {
-        return <AccountsMain accountsInfo={accountsInfo} current_user={current_user_info && current_user_info.name} groups={groupsExtraInfo || []} />;
-    } else {
+        return (
+            <AccountsMain
+                accountsInfo={accountsInfo}
+                current_user={current_user_info && current_user_info.name}
+                groups={groupsExtraInfo || []}
+                isGroupsExpanded={isGroupsExpanded}
+                setIsGroupsExpanded={setIsGroupsExpanded}
+            />
+        );
+    } else if (path.length === 1) {
         return (
             <AccountDetails accounts={accountsInfo} groups={groupsExtraInfo || []} shadow={shadow || []}
                             current_user={current_user_info && current_user_info.name} user={path[0]} />
         );
-    }
+    } else if (path.length === 2 && path[0] === "group" && groupsExtraInfo.length > 0 && accountsInfo.length > 0) {
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-18136-20230124-145550-0f2cd015-fedora-37-devel//Coverage/lcov/github-pr.diff.gcov.html)

Reply:
Same here.

Reply:
@martinpitt This is definitely tested. They is essentially the path for the groups details page, there is coverage for this. 

Did you ever spot test coverage being reporting wrong things in the past?